routes.js

```diff
@@ -0,0 +1,91 @@
+const express = require("express")
+const router = express.Router()
+const low = require('lowdb');
+// const path = require('path')
+const fileAsync = require('lowdb/lib/storages/file-async');
+const db = low('db/db.json', {
+  storage: fileAsync
+});
+
+module.exports = router;
+
+router.get('/movies', function(req, res) {
```

Comment:
Because all of the routes start with `/movies` in this router. I would prefer to see you mount the whole router like this: `app.use("/movies", router)` in your `app.js`.

Then here this route would look like this `app.get("/", ...` etc.